### PR TITLE
Small fixes in processor docs

### PR DESCRIPTION
**Pull request by lovromazgon**
### Description

Just some wording changes and formatting.

### CODE REVIEW

**Review by raulb — APPROVED** [review]

File: pkg/plugin/processor/builtin/impl/webhook/http.go [hunk 1]
```diff
@@ -88,7 +84,7 @@ func NewHTTPProcessor(l log.CtxLogger) sdk.Processor {
 func (p *httpProcessor) Specification() (sdk.Specification, error) {
 	return sdk.Specification{
 		Name:    "webhook.http",
-		Summary: "HTTP webhook processor",
+		Summary: "Trigger a HTTP request for every record.",
```
Comment (maha-hajja) [inline pkg/plugin/processor/builtin/impl/webhook/http.go:87]:
```suggestion
		Summary: "Trigger an HTTP request for every record.",
```

File: pkg/plugin/processor/builtin/internal/exampleutil/specs/webhook.http.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
   "specification": {
     "name": "webhook.http",
-    "summary": "HTTP webhook processor",
+    "summary": "Trigger a HTTP request for every record.",
```
Comment (maha-hajja) [inline pkg/plugin/processor/builtin/internal/exampleutil/specs/webhook.http.json:4]:
```suggestion
    "summary": "Trigger an HTTP request for every record.",
```

File: pkg/plugin/processor/builtin/impl/avro/decode.go [hunk 3]
```diff
@@ -108,7 +108,7 @@ func NewDecodeProcessor(logger log.CtxLogger) sdk.Processor {
 func (p *decodeProcessor) Specification() (sdk.Specification, error) {
 	return sdk.Specification{
 		Name:    "avro.decode",
-		Summary: "Decodes a field's raw data in the Avro format",
+		Summary: "Decodes a field's raw data in the Avro format.",
```
Comment (hariso) [inline pkg/plugin/processor/builtin/impl/avro/decode.go:111]: Nitpick: didn't we say summaries should NOT end in dots, since they're shown as headers?

Comment (lovromazgon) [reply pkg/plugin/processor/builtin/impl/avro/decode.go:111]: That's the case for example summaries, not specifications, these are shown below the header 🙈

**Pull request merged**